src/IdentityManager/IdentityManager.sln

```diff
@@ -0,0 +1,39 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Please move SLN file to repository root. On long period this will save good amount of time of launching project.